eldritch/events.py

```diff
@@ -2619,11 +2619,17 @@ def resolve(self, state):
         return False
 
     if self.seal_choice is None:
-      if self.character.clues < 5:  # TODO: this can also have modifiers
+      difficulty = 5
+      difficulty += state.ancient_one.get_modifier(self, "difficulty")
```

Comment:
Let's not name this difficulty - there is already a difficulty for the gate close attempt fight/lore check.
Maybe clues? seal_clues? Something like that.

Reply:
Done.